tests/test_tags.py

```diff
@@ -601,7 +580,7 @@ def test_linux_platforms_manylinux2014_armv6l(self, monkeypatch):
     def test_linux_platforms_not_manylinux_abi(
         self, monkeypatch, machine, abi, alt_machine
     ):
-        monkeypatch.setattr(tags, "_is_manylinux_compatible", lambda name, _: True)
+        monkeypatch.setattr(tags, "_is_manylinux_compatible", lambda name, _: False)
```

Comment:
This is a weird test. Previously it depended on the monkeypatch on line 42 to set `_have_compatible_glibc` in order to reject any glibc value.

Reply:
Do you have something specific you want to change that you haven't and so the comment is on its previous state? Or is there something you want to do that you haven't yet?

Reply:
Since I refactored `_have_compatible_glibc` the test is more reasonable. Previously `_have_compatible_glibc` did two things, so refactored it out into the component actions did two things, so refactored it out into the component actions